newIDE/app/src/UI/Theme/i18n.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+import { Style, stash } from './domain';
+
+const withDirection = Style({
+  get direction() {
+    return stash('language-ltr') ? 'unset' : 'rtl';
+  },
+});
+
+export const text = {
+  root: withDirection('root'),
+  label: withDirection('label'),
+};
+
+const withOrder = Style({
+  get order() {
+    return stash('language-ltr') ? 'unset' : (styles.order?.rtl ?? 100);
+  },
+});
+export const icon = {
+  root: withOrder('root'),
```

Comment:
This seems complex. Let's remove `Style` and just declare an inline function where needed.